ntbs-service/Migrations/20210217120522_DotNetUpgrade_Blank.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class DotNetUpgrade_Blank : Migration
```

Comment:
Adding this migration was a good idea - I think a little more explanation of why it's here would be good. (The name of the migration may make it obvious to us now, but may not make sense out of context a year from now).